demos/instr.cc

```diff
@@ -43,7 +43,9 @@ static void MFence() {
     defined(_M_IX86)
   _mm_mfence();
 #elif defined(__aarch64__)
-  asm volatile("dsb sy");
+  asm volatile(
```

Comment:
This really doesn't map to `mfence` anymore, and it also means that the `LFence(); MFence();` sequence we have in `ReadLatency` ends up inserting:
```asm
dsb
dsb
isb
```

This is okay as a temporary patch, but I expect we're going to end up writing different inline asm versions of `ReadLatency` for each architecture.

Reply:
I added an equivalent isb instruction also to the LFence.